bazel/canbench.bzl

```diff
@@ -30,6 +31,7 @@ def rust_canbench(name, results_file, **kwargs):
 
     canbench_bin = "$(location @crate_index//:canbench__canbench)"
     wasm_path = "$(location :{name}_wasm)".format(name = name)
+    pocket_ic_bin = "$(rootpath //rs/pocket_ic_server:pocket-ic-server)"
```

Comment:
We should use the latest server release: https://github.com/dfinity/pocketic/releases/tag/6.0.0

Reply:
You mean download https://github.com/dfinity/pocketic/releases/download/6.0.0/pocket-ic-x86_64-linux.gz? Canbench is already doing that and we wanted to avoid it (see the first thread in this PR). I think it would be best to provide `pocket-ic-server` as a crate and depend on a crate version.

Reply:
> Canbench is already doing that and we wanted to avoid it (see the first thread in this PR). 

sure, but we could save the bandwidth by using the bazel cache